examples/utils/example-utils/src/modelLoader/modelLoader.ts

```diff
@@ -99,6 +100,7 @@ export class ModelLoader<ModelType> implements IModelLoader<ModelType> {
 			url: modelUrl,
 			headers: { containerRef: container },
 		};
+		// eslint-disable-next-line import/no-deprecated
```

Comment:
Probably long-term solution here would be to have ModelContainerRuntimeFactory run createModel in response to a call on the entryPoint (rather than in response to a request).  Something like:

```ts
interface IModelContainerRuntimeEntryPoint<ModelType> {
    readonly createModel: (container: IContainer) => Promise<ModelType>;
}
```

Probably means that createModel needs partial application of the runtime arg when initializing the entryPoint.